src/openrct2-ui/WindowManager.cpp

```diff
@@ -247,7 +247,7 @@ class WindowManager final : public IWindowManager
                 uint32_t type = intent->GetUIntExtra(INTENT_EXTRA_LOADSAVE_TYPE);
                 std::string defaultName = intent->GetStringExtra(INTENT_EXTRA_PATH);
                 loadsave_callback callback = reinterpret_cast<loadsave_callback>(
```

Comment:
This type is only used in a few places. Would you mind renaming it `LoadSaveCallback` to follow current code style conventions?